patchwork-gui/src/main/java/com/patchworkmc/mixin/gui/MixinKeyboard.java

```diff
@@ -74,20 +75,26 @@ private void postKeyEvent(int i, boolean[] bls, ParentElement element, int key,
 	}
 
 	@Inject(method = "method_1458", at = @At("HEAD"), cancellable = true)
-	private static void charEvent(Element element, int character, int mods, CallbackInfo info) {
+	private static void method_1458(Element element, int character, int mods, CallbackInfo info) {
```

Comment:
Change all the mixin names in this class to something meaningful, like `hookCharTypedEventPre`